src/Lykke.Job.CandlesHistoryWriter.Services/Candles/CandlesCacheInitalizationService.cs

```diff
@@ -117,7 +117,12 @@ private async Task CacheAssetPairCandlesAsync(AssetPair assetPair, DateTime now,
                     foreach (var timeInterval in Constants.StoredIntervals)
                     {
                         var candles = await _candlesHistoryRepository.GetExactCandlesAsync(assetPair.Id, timeInterval, priceType, now, _amountOfCandlesToStore);
-                        await _candlesCacheService.InitializeAsync(assetPair.Id, priceType, timeInterval, candles.ToArray(), slotType);
+
+                        if (!candles.Any()) 
+                            continue;
+                        
+                        _log.Info(nameof(InitializeCacheAsync),$"{priceType} {timeInterval} {assetPair.Id} candles to cache = {candles.Count}");
```

Comment:
On more thing that I didn't saw first time:

```suggestion
                         _log.Info($"{priceType} {timeInterval} {assetPair.Id} candles to cache = {candles.Count}");
```